### DISCUSSION TITLE
Character Limits

### DISCUSSION THREAD

**Discussion by mikethediyman** [AIStoryBuilders/AIStoryBuildersOnline#4] [General]
Is there a limit on how many text characters you can paste into AI Storybuilder ?
I am only using it for the first time and found even the Tiltle ( Blossoming Love in Rosewood ) would not fully paste
Then when I tried to paste one page of text - it only seemed to accept approx two thirds of it - then even trying to continur typing wasn't possible Any advice would be much appreciated

**Comment by ADefWebserver** [comment]
Can we look the two different issues:

- Could not paste Title - There is a limit to how long your title can be
- Could not paste content in a paragraph - There is a limit to how much you can paste in a single paragraph. You can only paste in a single paragraph, not an entire page. Also you can only do this on The Chapters tab

**Comment by mikethediyman** [comment]
Thanks for your speedy response 
What is the actual limit for both above ?

**Comment by ADefWebserver** [comment]
* Story Title - 20 characters
* Single Paragraph - 5000 Characters

**Reply by mikethediyman** [reply]
Thanks so much for clarifying this for me.
